applications/GeoMechanicsApplication/custom_conditions/Pw_point_flux_condition.hpp

```diff
@@ -0,0 +1,67 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//
+//  Main authors:    Mohamed Nabi
+//                   John van Esch
+//
+
+#pragma once
+
+#include "custom_conditions/Pw_condition.hpp"
+#include "includes/serializer.h"
+
+namespace Kratos
+{
+
+template< unsigned int TDim, unsigned int TNumNodes >
+class KRATOS_API(GEO_MECHANICS_APPLICATION) PwPointFluxCondition : public PwCondition<TDim,TNumNodes>
+{
+public:
+    using GeometryType = Geometry<NodeType>;
+    using PropertiesType = Properties;
+    using NodesArrayType = GeometryType::PointsArrayType;
+    
+    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(PwPointFluxCondition);
+
+    PwPointFluxCondition();
+
+    PwPointFluxCondition(IndexType NewId, GeometryType::Pointer pGeometry);
+
+    PwPointFluxCondition(IndexType NewId, GeometryType::Pointer pGeometry, PropertiesType::Pointer pProperties);
+
+    ~PwPointFluxCondition() override;
```

Comment:
If I am not mistaken, Anne told that usually 'override' is not used with a destructor.

Reply:
I removed the destructure, as it is default. Do you think it is ok?

Reply:
Anne uses the following ~PwCondition() override = default; Perhaps, the best is to have the same approach. 'default' will force to create a default destructor.